### PR TITLE
revert: Change from fluentd creating s3 policy to manually creating policy with deployment hook

### PR DESCRIPTION
**Pull request by rtfpessoa**
Tested in sandbox using https://github.com/codacy/chart/tree/sandbox/revert-fluentpolicy and it fixed the issue.

### CODE REVIEW

File: codacy/templates/fluentd/lifecycle-police-job.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{{ if .Values.fluentdoperator.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ .Release.Name }}-fluentd-lifecycle-policy-creator"
+  labels:
+    {{- include "codacy.labels" . | indent 4 }}
```
Comment (afsmeira) [inline codacy/templates/fluentd/lifecycle-police-job.yaml:7]: Watch out for the character limit. Maybe you need to truncate.

File: codacy/templates/fluentd/lifecycle-police-job.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    # job is considered part of the release.
+    "helm.sh/hook": post-install,post-upgrade
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    metadata:
+      name: "{{ .Release.Name }}"
+      labels:
+        {{- include "codacy.labels" . | indent 8 }}
```
Comment (afsmeira) [inline codacy/templates/fluentd/lifecycle-police-job.yaml:19]: Here as well.

File: codacy/templates/fluentd/lifecycle-police-job.yaml (hunk 1 above)
Comment (rtfpessoa) [reply codacy/templates/fluentd/lifecycle-police-job.yaml:7]: Not here this is the list

Comment (rtfpessoa) [reply codacy/templates/fluentd/lifecycle-police-job.yaml:7]: I truncated the value in the helper in the other pr. This is just the usage of the other values

**Pull request merged**